Dockerfile

```diff
@@ -1,17 +1,64 @@
-FROM golang:rc
+## build Oragono
+FROM golang:rc-alpine AS build-env
 
-EXPOSE 6667/tcp 6697/tcp
+RUN apk add --no-cache git
```

Comment:
Any reason not to put them all on one line?
```suggestion
RUN apk add --no-cache git make curl
```

Reply:
Nah this makes sense to do, fix'd